From a681a79472df85cdc8c052bc61fb8b9e7b9a6495 Mon Sep 17 00:00:00 2001
From: Tim Orling <tim.orling@konsulko.com>
Date: Sun, 27 Feb 2022 11:50:36 -0800
Subject: [PATCH] setup.py: switch from distutils to setuptools

In Python 3.10, distutils is deprecated and slated for removal in Python
3.12. It also prevents 'setup.py bdist_wheel' from building a wheel.

Upstream-Status: Submitted
[https://github.com/systemd/python-systemd/pull/103]

Signed-off-by: Tim Orling <tim.orling@konsulko.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 099819b..fe9c03a 100644
--- a/setup.py
+++ b/setup.py
@@ -1,5 +1,5 @@
 import sys, os
-from distutils.core import setup, Extension
+from setuptools import setup, Extension
 from subprocess import Popen, PIPE, check_output
 
 def call(*cmd):
-- 
2.30.2

